TestFailer/TestFailer/src/misaligned.cpp

```diff
@@ -3,19 +3,20 @@
 
 namespace misaligned
 {
-    int computeColorCombination(int i, int j)
+    constexpr auto MAX_NUMBER_OF_COLORS = 5;;
+    int computeColorCombination(int majorColorIndex, int minorColorIndex)
```

Comment:
Function name is `computeColorCombination`, but it's computing an integer?
Is it computing the zero-based row index in the manual?